test/extensions/tracers/xray/config_test.cc

```diff
@@ -24,7 +24,7 @@ TEST(XRayTracerConfigTest, XRayHttpTracerWithTypedConfig) {
     name: envoy.tracers.xray
     typed_config:
       "@type": type.googleapis.com/envoy.config.trace.v2.XRayConfig
-      daemon_endpoint: 127.0.0.1
+      daemon_endpoint: 127.0.0.1:2020
```

Comment:
is the port required? Pretty sure App Mesh sets a default anyway. Just making sure :)

Reply:
Port is required by the function I'm using to parse the ip